worlds/Files.py

```diff
@@ -41,7 +41,7 @@ def get_handler(file: str) -> Optional[AutoPatchRegister]:
 
 class AutoPatchExtensionRegister(abc.ABCMeta):
     extension_types: ClassVar[Dict[str, AutoPatchExtensionRegister]] = {}
-    required_extensions: List[str] = []
+    required_extensions: ClassVar[Tuple[str]] = ()
```

Comment:
This `ClassVar` change isn't right.

A class is an instance of the metaclass.
So an instance variable of the metaclass is a class variable in the class.

The instances of `AutoPatchExtensionRegister` have the instance variable `required_extensions`.
Because those instances are classes, they are class variables in those classes.

That's why I pointed out line 377 when I talked about `required_extensions`

Here it would be `Tuple[str, ...]`
In line 377 it would be `ClassVar[Tuple[str, ...]]`

Reply:
Ah, I see. That's on me for not reading close enough.